src/NimBLEUtils.cpp

```diff
@@ -355,9 +356,13 @@ const char* NimBLEUtils::returnCodeToString(int rc) {
             return "Pairing over the LE transport failed - Pairing Request sent over the BR/EDR transport in process.";
         case  (0x0500+BLE_SM_ERR_CROSS_TRANS ):
             return "BR/EDR Link Key generated on the BR/EDR transport cannot be used to derive and distribute keys for the LE transport.";
-            
         default:
             return "Unknown";
+#else // #endif // #ifndef NIMBLEUTILS_NO_STRINGS
+        default:
+            return "No error texts";
```

Comment:
We could just return an empty string here.

Reply:
Yes, or a suggestion that they can be enabled at compile time?

Reply:
I'd rather an empty string for cleaner logs, or "<N/A>" at most.